File: 1397.py
while True:
    pa = pb = 0
    n = int(input())
    if n == 0:
        break
    for i in range(0, n):
        a, b = input().split(' ')
        a=int(a);b=int(b)
        if a == b:
            pa+=0
        elif a>b:
            pa+=1
        elif a<b:
            pb+=1
    print(pa,pb)

File: 1397.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
while(true){
    let pa = 0;let pb = 0;
    let n = Number(lines.shift());
    if(n === 0){
        break;}
    for(var i=0; i<n; i++){
        ba = lines.shift().split(' ')
        let a=parseInt(ba[0]);let b=parseInt(ba[1])
        if(a == b){
            pa+=0}
        else if(a>b){
            pa+=1}
        else if(a<b){
            pb+=1}}
    console.log(pa,pb)
}
